flink-runtime-web/web-dashboard/app/scripts/common/filters.coffee

```diff
@@ -28,6 +28,32 @@ angular.module('flinkApp')
 
   amDurationFormatExtendedFilter
 
+.filter "humanizeDuration", ->
```

Comment:
Sorry that I've asked about this. As a committer I have to make sure we properly mark copied code, and this functionality is something very commonly used, and I'm sure there are plenty of implementations of this around in the web.
To be honest, I would have probably copied it from somewhere. That's why I was checking, just to make sure.